pkg/apis/policy/register.go

```diff
@@ -48,6 +49,8 @@ func addKnownTypes(scheme *runtime.Scheme) error {
 	scheme.AddKnownTypes(SchemeGroupVersion,
 		&PodDisruptionBudget{},
 		&PodDisruptionBudgetList{},
+		&extensions.PodSecurityPolicy{},
+		&extensions.PodSecurityPolicyList{},
```

Comment:
Did you mean to register the extensions one here?

Reply:
This is internal, which hubs to the types located in extensions (same as other types that are moving between API groups)